### DISCUSSION TITLE
a question

### DISCUSSION THREAD

**Discussion by nahiassis** [HaxeFlixel/flixel#2476] [Q&A] [answered]
- **Haxe version:** 4.2.4
- **Flixel version:** 4.10.0
- **OpenFL version:** 9.1.0
- **Lime version:** 7.9.0
- **Targets:**
Windows
________________________________

its nothing from any state im trying to make or whatever
its a question
how do i use _customSoundTray from FlxGame
and if possible, can anyone give me an example of how to do it?

**Comment by nahiassis** [comment, marked as answer]
nvm i figured the stuff out

**Reply by Geokureli** [reply]
can you post the answer here for others?
